lib/pages/forward/search_forward_controller.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:debounce_throttle/debounce_throttle.dart';
+import 'package:flutter/material.dart';
+import 'package:matrix/matrix.dart';
+
+class SearchForwardController {
```

Comment:
Why do you have to create another controller, i have `SearchController` already, please use it otherwise reuse it

Reply:
because, this search not only search for `Contacts` also search for `chat`, IMO

Reply:
> because, this search not only search for `Contacts` also search for `chat`, IMO

Exactly, It only handles the search part of the forward

Reply:
yes, we should more details on the search topic later (it is not in this ticket)